.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,28 @@
+name: Coding Standards
+
+on:
+  push:
+    branches:
+      - main
+      - v2.**
```

Comment:
```
- v[0-9].[0-9]
```

Reply:
I've used the same as in https://github.com/predis/predis/blob/113145f3065b8f34da2e09359f403e4db213b136/.github/workflows/tests.yml#L7

should I change it also there?

Reply:
This is fine 👍